templates/main.php

```diff
@@ -167,46 +167,46 @@ class="collection"
             <div class="app-navigation-entry-menu bubble sortorder-dropdown">
```

Comment:
If you want to work on this in another pr, I will change my review to approved to merge this fast! ;)